apps/www/_blog/2023-08-08-supabase-local-dev.mdx

```diff
@@ -0,0 +1,309 @@
+let movie: Database['public']['Tables']['movies']['Row'] = // ...
+
+// After
+let movie: Tables<'movies'> = // ...
+```
+
+<summary>
+<details>- See full command<details>
+    
+    ```markdown
+    supabase gen types
+    Generate types from Postgres schema
+    
+    Usage:
+      supabase gen types [command]
+    
+    Available Commands:
+      typescript  Generate types for TypeScript
+    ```
+    </summary>
+
+## Official GitHub Action
+
+![Support for GitHub Actions](/images/blog/launch-week-8/day-2/support-for-github-actions.png)
+
+We’ve developed an official [GitHub Action](https://github.com/marketplace/actions/supabase-cli-action) which leverages the CLI. You can [generate types on every PR](https://supabase.com/docs/guides/cli/github-action/generating-types), or run [your tests on every commit](https://supabase.com/docs/guides/cli/github-action/testing).
+
+## Local Logging and Debugging
+
+Logs are now accessible locally in the Dashboard. Last launch week we released an open source logging server, with support for BigQuery. In the past few months we’ve added Postgres support to this server. This means that all of your local logs are accessible with no additional config - simply run supabase start and then visit the local dashboard to start debugging.
+
+![Logs are now accesible locally in the dashboard](/images/blog/launch-week-8/day-2/local-logging-debugging.png)
+
+## Stable releases
+
+We’ve moved the Supabase CLI to a fortnightly stable-release cycle. Every 2 weeks, we will update the `[latest](https://www.npmjs.com/package/supabase?activeTab=versions)` tag on npm, the `[supabase/tap](https://github.com/supabase/homebrew-tap/blob/main/supabase.rb)` for homebrew, and the `[supabase](https://github.com/supabase/homebrew-tap/blob/main/supabase.rb)` scoop bucket. You can find the binary downloads in our GitHub [latest release](https://github.com/supabase/cli/releases/latest).
+
+For the adventurous feature hunters, we’ve added a `beta` release channel for the CLI, with new releases on every PR merged. You can follow [this guide](https://github.com/supabase/cli#install-the-cli) to install Supabase CLI (beta).
+
+## Branching and Preview Environments
+
+And finally, probably our most anticipated feature - branching:
+
+![Branch manager in the Supabase dashboard](/images/blog/launch-week-8/day-2/branching-preview-environments.png)
+
+![Searching branches with Supabase](/images/blog/launch-week-8/day-2/branching-02.png)
+
+We’ve made big improvements to our local development above - but we have bigger ambitions. For several months we’ve been developing Supabase branching. 
+
+Branching in Supabase isn’t simply a database, it’s an entire backend - everything from your Postgres database to your [50GB videos](https://supabase.com/blog/storage-v3-resumable-uploads). It drastically improves the experience of managing environments so that developers and teams can spend less time on DevOps and more time building their ideas faster.
```

Comment:
```suggestion
Supabase isn’t simply a database, it’s an entire backend - everything from your Postgres database to your [50GB videos](https://supabase.com/blog/storage-v3-resumable-uploads). Branching improves the experience of managing environments so that developers and teams spend less time on DevOps and more time building.
```